contrib/templates/data-engineering/assets/etl-pipeline/template/assets/{{.pipeline_name}}/{{.pipeline_name}}.job.yml.tmpl

```diff
@@ -0,0 +1,24 @@
+# The job that triggers {{.pipeline_name}}.
+resources:
+  jobs:
+    {{.pipeline_name}}_job:
+      name: {{.pipeline_name}}_job
+
+      trigger:
+        # Run this job every day, exactly one day from the last run; see https://docs.databricks.com/api/workspace/jobs/create#trigger
+        periodic:
+          interval: 1
+          unit: DAYS
+
+      {{- if not is_service_principal}}
+
+      email_notifications:
```

Comment:
email notifications in dev mode are annoying. 

We should either turn them off on dev mode (@pietern )
Or move this into a prod override

```
targets:
 dev:
   jobs:
     {{.pipeline_name}}_job:
       email_notifications:
          ...
```

Reply:
Will use variables instead!